templates/parachain/pallets/template/Cargo.toml

```diff
@@ -22,7 +22,7 @@ scale-info = { version = "2.11.1", default-features = false, features = [
 
 # frame deps
 frame-benchmarking = { path = "../../../../substrate/frame/benchmarking", default-features = false, optional = true }
-frame-support = { path = "../../../../substrate/frame/support", default-features = false }
+frame-support = { path = "../../../../substrate/frame/support", default-features = false, features = ["experimental"] }
```

Comment:
We should not add experimental stuff to the template. The point of it being that it is still experimental and subject to change.

Reply:
So revert the changes regarding construct runtime v2? @gupnik

Reply:
> So revert the changes regarding construct runtime v2? @gupnik

It's not experimental anymore. You should be able to use it without that flag @Daanvdplas.

Reply:
Then we should not enable the experimental feature :P

Reply:
Apologies, I thought seeing it being used in the kitchen sink runtime but I think I had a bad dream. Thanks Kian